assets/src/components/button/style.scss

```diff
@@ -0,0 +1,70 @@
+/**
+ * Button styles,
+ */
+
+.newspack-button {
+	.components-button {
```

Comment:
If we get rid of the wrapper `div`, we can get rid of the `.components-button` selector and apply all styles to `.newspack-button`

Reply:
We're inheriting styles from `.components-button` rather than duplicating here, so removing the selector will break our styling a bit.

Reply:
See [this comment](https://github.com/Automattic/newspack-plugin/pull/46/files#r283315652)